backend/routes/jobPostRouters.ts

```diff
@@ -33,6 +33,19 @@ jobPostRouter.get(
   }
 );
 
+jobPostRouter.get(
+  "/manager",
```

Comment:
What about adding `managerId` to the supported filters for GET /jobPost? It makes more sense to me as a path, and then you don't even need getJobPostingsWithManagerId. i see now that Dima suggested this on the design doc